Cloud Application Manager/Administering Your Organization/resource-tags.md

```diff
@@ -2,6 +2,7 @@
 "title": "Tag Instances",
 "date": "10-30-2017",
 "author": "",
+"keywords": ["tags", "settings", "organization tag", "organization tag"],
 "attachments": [],
 "contentIsHTML": false
 }}}
```

Comment:
Please, use standard structure: TOC, sections, etc.

Reply:
@gsanchezu thanks for the point, I added it for Tags and also Webhook that hadn't TOC section.